skietbaan_fe/src/components/NavbarMenu.js

```diff
@@ -1,41 +1,127 @@
 import React, { Component } from 'react';
-import { Navbar, NavDropdown, Form, Nav, Container } from 'react-bootstrap';
+import { Navbar } from 'react-bootstrap';
 import '../scss/navbar.css';
 import 'font-awesome/css/font-awesome.min.css';
 import '../bootstrap/NavbarMenuStyle.css';
-
 class NavbarMenu extends Component {
+	constructor(props) {
+		super(props);
+
+		this.isActive = this.isActive.bind(this);
+		this.isHome = this.isHome.bind(this);
+		this.isCreate = this.isCreate.bind(this);
+		this.isScoreCapture = this.isScoreCapture.bind(this);
+		this.isProfile = this.isProfile.bind(this);
+		this.isDocuments = this.isDocuments.bind(this);
+		this.isNotifications = this.isNotifications.bind(this);
+	}
+
+	isActive(route) {
+		if (window.location.pathname.endsWith(route)) {
+			return "nav-active";
+		}
+		else {
+			return "";
+		}
+	}
+
+	isHome(){
```

Comment:
Add a description to your alt when inserting the image tag so that you can provide better context to search engines crawlers.
Do the same for all the image tags

Reply:
I put descriptions in